### ISSUE TITLE
lower mIoU than the result of the paper

### ISSUE THREAD

**Issue by MinWang1997** [TUI-NICR/ESANet#25]
Hi,
I used the default setting to train NUYv2 and the result 47.7 is much lower than the result of the paper 50.3.

Does anyone get a similar result as follows?
mIoU test kv1: 0.4748908361165817
mIoU test: 0.4748908361165817
0.4748908361165817

What do you think is the reason for this?

`# or use the default arguments
python train.py \
    --dataset nyuv2 \
    --dataset_dir ./datasets/nyuv2 \
    --pretrained_dir ./trained_models/imagenet \
    --results_dir ./results`

**Comment by danielS91** [comment]
Note that the labels in NYUv2 are not perfect and the results may vary a bit. Therefore, we repeated each training serveral times and used a grid search for the learning rate ranging from 0.005 to 0.04 for NYUv2. Finally, we picked the highest value.
